### DISCUSSION TITLE
Welcome to awesome-cyber-skills Discussions!

### DISCUSSION THREAD

**Discussion by joe-shenouda** [joe-shenouda/awesome-cyber-skills#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MissDarlingNicki** [comment]
My name is Nicki, and I have absolutely NO IDEA what I'm doing here... 😬😉😅🥳😏

Yet. 🤔

**Reply by joe-shenouda** [reply]
Welcome, Nicki! Feel free to explore some links and begin your journey. Remember, every remarkable accomplishment has a starting point.

**Comment by Gotit4DaLo** [comment]
Hi Nikki welcome so what do you know about the fact cuz apparently you're here for a reason aren't we not reply or nothing but welcome to the day we're here if you need us

**Comment by BatoniNikolozi** [comment]
Hey, I'm starting using Kali Linux, any advices?
